experimental/rsocket-src/RSocketRequester.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include "rsocket/RSocketRequester.h"
+
+using namespace folly;
+
+namespace rsocket {
+
+std::shared_ptr<RSocketRequester> RSocketRequester::create(
+    std::unique_ptr<StandardReactiveSocket> srs,
+    EventBase& eventBase) {
```

Comment:
I started out trying to make it Executor and found that Executor didn't have the methods I was looking for. I'll put the TODO.